# Undergraudate Research Visitor at The University of Hong Kong

Research Visitor supervised by lab PI Dr. Juan Diego Gaitan Espitia [https://www.scifac.hku.hk/people/gaitnespitia-juan-diego-jd] and Ecology Ph.D student Coskun Guclu. Completed this summer research placement from July 2021 to October 2021 (Undergraduate First Year Summer).

**Project:** Carried out independent research in molecular phylogeny in Salicaceae sensu stricto (s.s.) genera Populus & Salix along with a Coskun Glucu. Personally developed a R code that takes data from the GBIF (Global Biodiversity Information Facility) database and maps the distribution of certain plant species. Benchmarked the parameters of the code and tested it with "Salix babylonica" which generated the distribution map found below. Developed this program to further understand the overall distribution of Salix and Poplus species, hoping to gain a better understanding of the geographical locations, leading to resolving the controversy between the molecular phylogeny of Salicaceae Populus and Salix species. Futhermore, helped constuct two phylogenetic tree hypotheses based on morphological and trait data gathered from (Skortsov, 1968) and (Eckenwalder, 1995) and molecular plastid data from (Zong et al, 2019). Used these trees as baselines for developing our phylogenetic tree consisting of 96 species of Salicaceae sensu lato (s.l), 88 species sensu stricto (s.s.), 1 species basal angiosperm. Whole genomes (.genbank)(.fasta) and Protein-coding genes including non-orthologous (.fasta) (range of 116-131 genes) were used to capsulate the entireity of the Salicaceae sensu stricto (s.s.) genera of Populs and Salix sepcies. 

* Contributions to this ongoing project results in a co-author position once the paper has been finialised and submitted. 

**Technical Aspect:** Extracted 50+ genomes of Salicaceae via (genbank) and performed data cleaning and analysis through Ugene and Geneious. Moreover, performed geographical data extraction through reading literature and research conducted on the evolutionary lineage of the selected species. Identified, extracted and concatenated orthologous protein coding genes for 97 species of salicaceae to fasta file (likely 77 based off of (Zhang et al, 2018)). Utilized HPC clusters to perform alignment (concatenated orthologous genes and whole genomes using HPC (geneious)) then generated evolutionary model (phylogenetic trees) by using maximum likelihood (ML) analysis and bayesian inference. Generation of phylogenetic trees and phylogeny findings interpretation were done by the PhD student. 

## Salix bablonica population distribution 
![WhatsApp Image 2021-08-05 at 10 20 50 AM](https://user-images.githubusercontent.com/83464019/172001274-4af6c30c-9b5a-47e9-9882-ce6b60ad64ea.jpeg)

### Figure 1. Morphological and Trait Data (Skortsov, 1968) and (Eckenwalder, 1995)
<img width="743" alt="Screenshot 2022-06-04 at 2 49 23 PM" src="https://user-images.githubusercontent.com/83464019/172003619-0d36fa2d-6aa6-4bdb-9c9a-e5d4e57a7242.png">

### Figure 2. Molecular Plastid Data (Zong et al, 2019)
<img width="738" alt="Screenshot 2022-06-04 at 2 49 28 PM" src="https://user-images.githubusercontent.com/83464019/172003645-19d1d9f1-8e0a-478e-a047-e6b78e1629c1.png">
